doc/notes/mapping/9337_mapping.md

```diff
@@ -0,0 +1,258 @@
+we will explore the relationship between elements in Requirements and
+elements in Design.  We will map them together into the Implementation.
+Design is complete when each element listed in the Requirements
+Specification has an architectural element in the Design that can be
+used to implement it.
+
+#### Models  
+Again, MDA prescribes modeling.  A list of elements can be sufficient
+for simple systems.  But for complex systems, a more precise definition
+of the Design can be communicated in a model.
+A Platform Specific Model (PSM) is crafted that illustrates the
+elements in the system/software architecture.  Since Design focuses
+on the platform and technologies, the model is made specific to the
+platform.  This Design model is independent of functional requirements
+but is very specific to the target architecture chosen technologies.
+The model adheres to
+semantics that clearly define and constrain each element in the Design.
+Each element of the system/software architecture is illustrated in a
+model of the software architecture.
+
+With a model of Requirements and a model of Design, a model-to-model (M2M)
+transformation can be done that maps these model elements together
+into the Implementation.
+As a final step of the Design process a set of rules is defined.
+One rule is defined for each type of element in the Requirements.
+Each rule explains how to pair an element in the Requirements with
+an element in the Design.
+This set of mapping rules is the core of the Model Translation.
+These rules can be applied manually.
+
+With models, much of this mapping and translation can be automated.
+
+## Implementation  
+Implementation realizes the Requirements within the constraints of the
+Design.
+
+In some systems, the process of Implementation is not clearly specified.
+In such cases the implementation team must satisfy the input Requirements
+and Design in an _ad hoc_ fashion.  A successful Implementation is proven
+by Testing.  But the Implementation team is not given a process whereby
+the Requirements and Design _translate_ into the Implementation.  Such
+a set of rules make Implementation easier, faster and more precise.
+
+Model Translation provides a more precise, repeatable and step-by-step
+approach to implementation.  The process of Implementation is accomplished
+by _Mapping_.  Elements from the Requirements Specification are paired
+with elements from the Design and realized in the Implementation.
+The rules to govern this mapping are written as part of the Design.
+
+## Example  
+Following is a simple example of Model Translation using Mapping.
+The example is a GPS Tracker used to track the location of a person
+or boat sailing on the sea.
+The example shows Requirements, Design and a Mapping to Implementation.
+
+### GPS Tracker  
+
+#### GPS Tracker:  Requirements  
+Consider a GPS Tracker that acquires and accumulates track points
+on a periodic time basis to record the path of the person or ship
```

Comment:
"track points" isn't clear, but I see why you want to introduce the term here. I'd use "location information (track points)"